src/tests/ftest/cart/test_group_np_srv.c

```diff
@@ -35,6 +35,37 @@
 #include "test_group_rpc.h"
 #include "test_group_np_common.h"
 
+/* Callback to process a SWIM message */
+static void
+swim_crt_event_cb(d_rank_t rank, enum crt_event_source src,
+		  enum crt_event_type type, void *arg)
+{
+	/* Example output for SWIM CRT_EVT_DEAD on rank #2:
+	 *	 rank = 2, crt_event_source = 1, crt_event_type = 1
+	 *
+	 *		enum crt_event_type {
+	 *			CRT_EVT_ALIVE,
+	 *			CRT_EVT_DEAD,
+	 *		};
+	 *		enum crt_event_source {
+	 *			CRT_EVS_UNKNOWN,
+	 *			CRT_EVS_SWIM,
+	 *		};
+	 */
+
+	D_DEBUG(DB_TEST, "Cart callback event: "
+		"rank = %d, "
+		"crt_event_source = %d, "
+		"crt_event_type = %d\n",
+		 rank, src, type);
+
+	if (src == CRT_EVS_SWIM) {
+		swim_status_by_rank[rank] = type;
+	}
+
+	return;
+}
```

Comment:
(style)  void function return statements are not generally useful

Reply:
8d067d7e14978128bc9546fc7f4cdb573d6e8639